### PR TITLE
[GOV] Code of Conduct revision

### LINKED ISSUE #35: [GOV] Changes to the code of conduct
Recent changes have already been reverted (#18 #17), but we should discuss if we want to make any further changes to the CoC.

### PR DESCRIPTION
**Pull request by MatthewMiddlehurst**
#### Reference Issues/PRs

Relies on #195
Resolves #35 

#### What does this implement/fix? Explain your changes.

This completely revises the current code of conduct in the style of #195. 

The 340 lines of text in the current document did not prevent any of the issues encountered previously. Having a more streamlined and condensed document which can get across the core principles of our community and contact points is more useful IMO.

The new version piggybacks off the general Python code of conduct for the most part https://www.python.org/psf/codeofconduct/.

### CODE REVIEW

**Review by TonyBagnall — DISMISSED** [review]
read through it all and it seems just right to me

**Comment by MatthewMiddlehurst** [comment]
I have updated some of the repo text files to give a bit of a summary rather than just linking the webpage.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
-You can find our Code of Conduct on our [website](https://www.aeon-toolkit.org/en/latest/get_involved/code_of_conduct.html).
+You can find our full Code of Conduct (CoC) on our [website](https://www.aeon-toolkit.org/en/latest/code_of_conduct.html).
+
+The code of conduct aims to ensure a positive and respectful experience for all
+members of the community. Community members should follow the principles set out in
+the [Python Software Foundation Code of Conduct](https://www.python.org/psf/codeofconduct/).
+
+If you want to make a CoC report, contact the general `aeon` email
+[contact@aeon-toolkit.org](mailto:contact@aeon-toolkit.org) or message a member of the
+[COCW](https://www.aeon-toolkit.org/en/latest/governance.html#code-of-conduct-workgroup)
```
Comment (TonyBagnall) [inline CODE_OF_CONDUCT.md:11]: need to give full name for COCW

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
 # Code of Conduct
 
-You can find our Code of Conduct on our [website](https://www.aeon-toolkit.org/en/latest/get_involved/code_of_conduct.html).
+You can find our full Code of Conduct (CoC) on our [website](https://www.aeon-toolkit.org/en/latest/code_of_conduct.html).
```
Comment (TonyBagnall) [inline CODE_OF_CONDUCT.md:3]: goes to 404 for me

**Review by TonyBagnall — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (MatthewMiddlehurst) [reply CODE_OF_CONDUCT.md:3]: This is the location post-move on the docs,

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (MatthewMiddlehurst) [reply CODE_OF_CONDUCT.md]: Done

**Review by TonyBagnall — APPROVED** [review]

**Pull request merged**